tests/test_external_search.py

```diff
@@ -321,7 +321,10 @@ def test__run_self_tests(self):
 
         eq_("Generating search document for the specified term: 'a search term'", test_results[1].name)
         eq_(True, test_results[1].success)
-        eq_("{'author': 'author', 'meta': {'id': 'id'}, 'id': 'id', 'title': 'Sample Book Title'}", test_results[1].result)
+        eq_(
+            '{\n "author": "author", \n "meta": {\n  "id": "id"\n }, \n "id": "id", \n "title": "Sample Book Title"\n}',
+            test_results[1].result
+        )
```

Comment:
This might be easier to test if you parsed `test_results[1].result` as JSON and then looked at the JSON object.